### ISSUE TITLE
Enhanced Loss Function based on Laplacian Eigenmaps for Graph Classification

### ISSUE THREAD

**Issue by 2nazero** [AIML-K/GNN_Survey#25]
```bib
@inproceedings{xiao2023enhanced,
  title={Enhanced Loss Function based on Laplacian Eigenmaps for Graph Classification},
  author={Xiao, Ye and Li, Ruikun and Vasnev, Andrey and Gao, Junbin},
  booktitle={2023 International Joint Conference on Neural Networks (IJCNN)},
  pages={1--8},
  year={2023},
  organization={IEEE}
}
```

**Comment by 2nazero** [comment]
<img width="824" alt="스크린샷 2024-11-11 오후 2 54 35" src="https://github.com/user-attachments/assets/ec27abff-91f7-4eb1-a704-7be59cceaaed">

이 논문에서는 그래프 신경망(GNN)의 그래프 분류 성능을 높이기 위한 새로운 손실 함수 (LEELoss)에 대해 말하고자 한다.

기존 손실 함수는 단순히 교차 엔트로피(cross-entropy) 손실로, 이는 분류 작업에서 모델이 예측한 확률과 실제 레이블 간의 차이를 줄이는 데 초점을 두었다면, 다음 논문에서는 라플라시안 고유지도 방법을 손실함수에 추가하여 그래프의 구조적 유사성까지 고려해 교차 엔트로피 손실에 라플라시안 고유지도 기반 정규화 항을 추가한 형태로 손실함수를 구성하였다.

<img width="358" alt="스크린샷 2024-11-11 오후 2 59 21" src="https://github.com/user-attachments/assets/934443f0-9c91-4907-9ca7-b282bea57ec5">
위 수식이 최종 손실 함수의 모습이다.

<img width="382" alt="스크린샷 2024-11-11 오후 3 03 44" src="https://github.com/user-attachments/assets/fc7929eb-77bc-423d-a662-e64819c45db7">
라플라시안 고유지도의 최적화 함수 수식의 형태이다. 수식을 보면 좌변에서는 인접한 노드 i와 j의 임베딩 y_i와 y_j간의 관계를 나타내고 있고, 우변에서는 노드 임베딩 행렬 Y에 대한 라플라시안 고유지고 손실을 계산한 값으로, 인접한 노드들이 비슷한 임베딩을 가지도록 유도하고 있다. 즉, 지역적 구조를 반영한 임베딩을 이용해 추후 손실함수를 계산하는 데 쓴다.
